benchmarks/src/garage_benchmarks/experiments/algos/her_garage_tf.py

```diff
@@ -42,7 +42,7 @@ def her_garage_tf(ctxt, env_id, seed):
     deterministic.set_seed(seed)
 
     with LocalTFRunner(ctxt) as runner:
-        env = GarageEnv(normalize(gym.make(env_id)))
+        env = normalize(GymEnv(gym.make(env_id)))
```

Comment:
```suggestion
        env = normalize(GymEnv(env_id))
```

Since the garage codebase is the largest source of usage examples, we should always make sure to use our own APIs in the most idiomatic way. The option of passing a string to GymEnv is there to make the code shorter/easier-to-read, so we should use it if we can.